lib/options.js

```diff
@@ -216,8 +216,8 @@ module.exports = optionator({
         },
         {
             option: "print-config",
-            type: "Boolean",
-            description: "Print the configuration to be used"
+            type: "path::String",
+            description: "Print the configuration which would be used for linting a given file (linting will not occur)"
```

Comment:
I'm guessing this will cause the text to wrap when output on the console. I'm not sure this needs to be changed from the previous text, but if it does, then can you make it a bit shorter?